src/librbd/operation/RenameRequest.h

```diff
@@ -57,6 +57,7 @@ class RenameRequest : public Request<ImageCtxT>
 protected:
   virtual void send_op();
   virtual bool should_complete(int r);
+  int filter_return_code(int r) const override;
```

Comment:
Just from aesthetic point of view I would use virtual instead of override for `filter_return_code` too. But this up to you.

Reply:
... the new(er) coding standards have us use "virtual" only for the base definition and "override" everywhere else.

Reply:
But I suppose this standard has not been backported to jewel  :-) Anyway, I am fine with it.

Reply:
... correct, but no reason to put any effort into using older standards when backporting.

Reply:
Agreed. I was just looking on this from the point of view of an occasional reader outside who happened  to dig through the old version, e.g. hunting a bug. When she see two functions declared as virtual and one as override she might think if there any hidden reason for this.